src/utils/strings/prose.js

```diff
@@ -0,0 +1,28 @@
+export function formatListAsProse(list, options = {}) {
+  const {
+    inclusive = true
+  } = options;
+
+  if (!Array.isArray(list)) {
+    return String(list);
+  }
+
+  const count = list.length;
+  const lastI = count - 1;
+  const conjunction = inclusive ? 'and' : 'or';
+  return list.reduce((acc, item, i) => {
```

Comment:
Too bad that there is no `slice` that doesn't copy array, otherwise we could simplify it like this :)
```
if (count <= 2) {
	return list.join(` ${conjunction} `);
}

return list.slice(0, -1).join(', ') + `, ${conjunction} ${list[count - 1]}`;
```